### PR TITLE
Search validaton on forms and Dashboard.

### PR DESCRIPTION
**Pull request by Josephalexantony-aot**
# Issue Tracking

JIRA: https://aottech.atlassian.net/browse/FWF-1972
Issue Type: BUG

# Changes

Added validation on serach box for removing special charactors. 

# How has the change been tested?

# Screenshots (if applicable)

# Build Success screenshot (Till a CICD pipeline is set up)

# Notes

### CODE REVIEW

**Comment by sreehari-aot** [comment]
@abhilash-aot @Josephalexantony-aot I am not sure, how is this a bug.

**Comment by AbijithS-aot** [comment]
please try to avoid regex patterns since some of regex will not load in safari browsers and will not work in ios devices. Please be aware of that. Use logics instead of regex

**Review by abhilash-aot — APPROVED** [review]

**Review by abilpraju-aot — APPROVED** [review]

File: forms-flow-web/src/components/Dashboard/Dashboard.js [hunk 1]
```diff
@@ -284,7 +284,7 @@ const Dashboard = React.memo(() => {
                           }
                           onChange={(e) => {
                             setShowClearButton(e.target.value);
-                            setSearchTextInput(e.target.value);
+                            setSearchTextInput(e.target.value.replace(/[^\w\s]/gi, ""));
```
Comment (abhilash-aot) [inline forms-flow-web/src/components/Dashboard/Dashboard.js:287]: Check if the regex works in all the browsers. We had similar issue before

Comment (Josephalexantony-aot) [reply forms-flow-web/src/components/Dashboard/Dashboard.js:287]: replace() is supported by all modern web browsers, including Internet Explorer, Edge, Chrome, Firefox, and Safari.

Comment (abhilash-aot) [reply forms-flow-web/src/components/Dashboard/Dashboard.js:287]: Check on ios device/ Safari . the regex pattern as such may have issue as well. The pattern format support may be different

**Pull request closed without merging**